_posts/2018-11-14-cockpit-182.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Cockpit 182
+author: Gundersanne
+date: 2018-11-14
+tags: cockpit linux
+slug: cockpit-182
+category: release
+summary: Cockpit with libvirt connection choice in VM creation and Software Updates improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 182.
+
+### Machines: Add libvirt connection choice in VM creation dialog
+
+The VM creation dialog now offers different connections to the libvirt
+daemon. The options result in either a user session URI or a system URI.
```

Comment:
Instead of "the libvirt daemon", consider either:

- "the libvirt service"
- "libvirt"

Reply:
Also, remove the duplicate "URI" and "a":

```suggestion
daemon. The options result in either a user session or system URI.
```